skeleton/.github/workflows/ci.yaml

```diff
@@ -13,11 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 11
       uses: actions/setup-java@v2
```

Comment:
could update the action versions while you're here too? v3 I think for both checkout and setup-java